### ISSUE TITLE
Provide a deps.edn for Clojure command line users

### ISSUE THREAD

**Issue by pmonks** [fn-fx/fn-fx#49]
It would be great if the project included a `deps.edn` file, so that users of the [Clojure command line tools](https://clojure.org/guides/deps_and_cli) can evaluate the library.

**Comment by jolby** [comment]
I've just tried using a deps.edn file (OSX, jdk 11, brew clojure 1.9.0.397), and it hangs. Looks like its getting stuck on resolving circular deps:
https://dev.clojure.org/jira/browse/TDEPS-79?focusedCommentId=50341&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel

**Comment by pmonks** [comment]
Good catch @jolby!  I added [a comment regarding classifiers](https://dev.clojure.org/jira/browse/TDEPS-79?focusedCommentId=50473#comment-50473) - that may be the issue here.

**Comment by pmonks** [comment]
Appears to be blocked on https://dev.clojure.org/jira/browse/TDEPS-79.

**Comment by zcaudate** [comment]
I don't understand. Can't the clj users just pull the library down from maven?

**Comment by pmonks** [comment]
@zcaudate yeah that's how `clj` works, but right now tools.deps (which `clj` is the front end of) seems to have two bugs:

1. it doesn't detect circular dependencies (so ends up spinlocking)
2. it doesn't appear to be sensitive to classifiers, which (in the case of `openjfx`) means it ends up synthesising a circular dependency that shouldn't exist, thereby running afoul of the previous point

These are what [TDEPS-79](https://dev.clojure.org/jira/browse/TDEPS-79) is tracking.

**Comment by zcaudate** [comment]
Ah. Got it.

**Comment by jdf-id-au** [comment]
Java 8's fine in the meantime.

`{:deps {fn-fx/fn-fx-javafx {:mvn/version "0.5.0-SNAPSHOT"}}}` works as expected, using:

- macOS 10.13.6
- java 1.8.0_192 via `brew cask install java8`
- clojure 1.9.0.391 also via `brew`

**Comment by MageMasher** [comment]
https://dev.clojure.org/jira/browse/TDEPS-79 is now resolved. I can confirm deps now works with the latest version of tools.deps
